### DISCUSSION TITLE
Экспорт

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#266] [Как это сделать?] [answered]
Могу ли я вывод документа при экспорте csv изменять, скобки квадратные убрать например, при печати шаблоны, а тут? 
![image](https://user-images.githubusercontent.com/75322878/201656233-6a52bdb0-5fad-4984-9259-7ad8cb1c4b5e.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Стандартный экспорт в `.csv` технический и не настраиваемый. Настраиваемые шаблоны, генерация `.xlsx` и прочие ништяки будут в пакете коммерческих сервисов за абонентскую плату. В течение месяца уже будет первая версия реализована.
